test/TestDataStorageWrite.java

```diff
@@ -0,0 +1,40 @@
+import factorapi.DataStorage;
+import factorapi.WriteResponse;
+import factorapi.WriteRequest;
+import org.mockito.Mockito;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
This is what is triggering the checkstyle error, I don't think it likes when you use the ".*" for imports, should probably change to specific ones